src/test/java/paymelah/logic/parser/AddCommandParserTest.java

```diff
@@ -90,18 +90,6 @@ public void parse_compulsoryFieldMissing_failure() {
         assertParseFailure(parser, VALID_NAME_BOB + PHONE_DESC_BOB + TELEGRAM_DESC_BOB + ADDRESS_DESC_BOB,
                 expectedMessage);
 
-        // missing phone prefix
-        assertParseFailure(parser, NAME_DESC_BOB + VALID_PHONE_BOB + TELEGRAM_DESC_BOB + ADDRESS_DESC_BOB,
-                expectedMessage);
-
-        // missing telegram prefix
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + VALID_TELEGRAM_BOB + ADDRESS_DESC_BOB,
-                expectedMessage);
-
-        // missing address prefix
-        assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + TELEGRAM_DESC_BOB + VALID_ADDRESS_BOB,
-                expectedMessage);
-
```

Comment:
Should move to `#parse_optionalFieldsMissing_success()`

Reply:
Rewrote these tests